tests/pytorch/test_dataloader.py

```diff
@@ -218,8 +217,8 @@ def _check_device(data):
 def test_node_dataloader():
     sampler = dgl.dataloading.MultiLayerFullNeighborSampler(2)
 
-    g1 = dgl.graph(([0, 0, 0, 1, 1], [1, 2, 3, 3, 4]))
-    g1.ndata['feat'] = F.copy_to(F.randn((5, 8)), F.cpu())
+    g1 = dgl.graph(([0, 0, 0, 1, 1], [1, 2, 3, 3, 4])).to(F.ctx())
```

Comment:
I think this change disable/skip the test of underlying graph copy from cpu to gpu within NodeDataLoader, as graph is not on the target device. After this change, graph is always on the target device already.

Reply:
if graph is on cpu when created and target device is gpu, the sampling will fail without this change?

Reply:
@Rhett-Ying Fixed in #3482